### PR TITLE
Added blog post for connected registry

### PR DESCRIPTION
**Pull request by toddysm**

### CODE REVIEW

**Comment by toddysm** [comment]
Abandoning this one. Will create one from a fork.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.vscode/
+samples/dotnetcore/image-transfer/ContainerRegistryTransfer/obj/
```
Comment (sajayantony) [inline .gitignore:2]: If you are adding this use this for reference - https://github.com/github/gitignore/blob/master/VisualStudio.gitignore#L31

File: docs/blog/connected-registry.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ms.date: 01/05/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Private Preview - ACR Connected Registry Feature
+We are announcing the private preview of Azure Container Registry (ACR) connected registry feature.
+
+The connected registry feature of ACR allows you to deploy a registry on your premises and synchronize images between the ACR and your premises. It brings the container images and OCI artifacts closer to your on-premise container workloads and increases their performance and reliability.
```
Comment (sajayantony) [inline docs/blog/connected-registry.md:14]: **Increases their performance** sounds like the containers will become faster. Might want to rephrase this to - increases artifact acquisition or download performance or something on those lines.
